docs/sql/mysql/create_push_server_schema.sql

```diff
@@ -87,5 +92,7 @@ CREATE INDEX `push_campaign_user_campaign` ON `push_campaign_user`(`campaign_id`
 CREATE INDEX `push_campaign_user_detail` ON `push_campaign_user`(`user_id`);
 
 CREATE INDEX `push_inbox_id` ON `push_inbox` (`inbox_id`);
-CREATE INDEX `push_inbox_user_app` ON `push_inbox` (`user_id`, `app_id`);
+CREATE INDEX `push_inbox_user_app` ON `push_inbox` (`user_id`);
```

Comment:
There is not `app` in the `push_inbox_user_app` index, is the index name correct?

Reply:
Thank you for catching this, issue for MySQL and Oracle. PostgreSQL was correct.